### DISCUSSION TITLE
How are apps chosen for the showcase on the sveltekit site?

### DISCUSSION THREAD

**Discussion by ivakhutinsky** [sveltejs/kit#8473] [General]
How are apps chosen for the showcase on the sveltekit site?

We’re building with Sveltekit over at Careswitch.com and would love to be included!

**Comment by ivakhutinsky** [comment]
@Rich-Harris ? 🥺

**Comment by angezanetti** [comment]
Up! I'm building 2 projects with SvelteKit :D

**Comment by dummdidumm** [comment]
There is no formalized process for this. That said, a few guidelines:
- should be a topic people can relate to (not too niche)
- should look "visually pleasing" (highly subjective, I know)
- has a decent lighthouse score (we know there's much more to great sites than this, but it's an unfortunate reality that people will fingerpoint at bad scores without being able to grasp nuance)

The current showcase is going to stay as-is for a while. We may be adding a dedicated showcases page so that more sites can appear on there. What we don't want though is to make this a free advertisement channel for all sorts of businesses.
